requirements/base.in

```diff
@@ -5,14 +5,20 @@
 
 alqtendpy==0.0.4
 arrow==0.12.1
+appdirs==1.4.3
 attrs==19.3.0
 bitstruct==6.0.0
+boto3==1.14.41.0
+boto3-stubs==1.14.41.0
```

Comment:
Looks like this is importing https://pypi.org/project/boto3-type-annotations/ instead of https://pypi.org/project/boto3-stubs/?  If so, I can handle the relocking etc.  Or switch the imports over